pkg/reconciler/eventlistener/resources/deployment.go

```diff
@@ -37,11 +37,28 @@ const (
 )
 
 var (
-	strongerSecurityPolicy = corev1.PodSecurityContext{
+	baseSecurityPolicy = corev1.PodSecurityContext{
 		RunAsNonRoot: ptr.Bool(true),
+		SeccompProfile: &corev1.SeccompProfile{
+			Type: corev1.SeccompProfileTypeRuntimeDefault,
+		},
 	}
 )
 
+func getStrongerSecurityPolicy(cfg *config.Config) corev1.PodSecurityContext {
+	securityContext := baseSecurityPolicy
+	if !cfg.Defaults.IsDefaultRunAsUserEmpty {
+		securityContext.RunAsUser = ptr.Int64(cfg.Defaults.DefaultRunAsUser)
+	}
+
+	if !cfg.Defaults.IsDefaultRunAsGroupEmpty {
+		securityContext.RunAsGroup = ptr.Int64(cfg.Defaults.DefaultRunAsGroup)
+		securityContext.FSGroup = ptr.Int64(cfg.Defaults.DefaultRunAsGroup)
```

Comment:
Is FSGroup and runAsGroup same ??

Reply:
I set them as the same, as i do not need change it for my use case. Should i make it configurable through a Defaults or wait for a specific use case?

Reply:
I would say wait for a specific case
we can add it whenever we have usecase

Reply:
agreed. I will keep it as it is with FSGroup set to RunAsGroup

Reply:
Oh wait my point is can we completly remove FSGroup or is it required ?

Reply:
I need it for my usecase to fullfill aks policy: https://github.com/Azure/azure-policy/blob/master/built-in-policies/policySetDefinitions/Kubernetes/PSPRestrictedStandard.json#L256

Reply:
> I set them as the same, as i do not need change it for my use case. Should i make it configurable through a Defaults or wait for a specific use case?

Shall we configure it so that if someone wants they can set to different value
as both FSGroup and RUnAsGroup functionality is different

Reply:
@savitaashture updated :)